charts/tornjak/templates/NOTES.txt

```diff
@@ -74,8 +74,22 @@ Universal Trusted Workload Identity Service has completed.
       }
 {{- end }}
 
-Tornjak Backend API: {{ include "tornjak.apiURL" . }}
+{{- if not .Values.tornjak.config.backend.ingress }}
 
+No Backend Ingress provided. 
+Use port forwarding to access the Backend port:
+      kubectl -n spire port-forward spire-server-0 10000:10000
```

Comment:
it'd be nice if the port number could be provided in a variable perhaps?

Reply:
The ports are currently hardcoded in Tornjak:
https://github.com/spiffe/tornjak/blob/6eb59e1c561ae959b9bc9efe731bf48a39e08a06/tornjak-backend/cmd/agent/agent.go#L69
and also for TLS and mTLS:
https://github.com/spiffe/tornjak/blob/6eb59e1c561ae959b9bc9efe731bf48a39e08a06/run_tornjak.sh#L63-L65
So unless me allow changing them during the image instantiation, there is no point of providing them as parameters here.